app/src/validators/note.ts

```diff
@@ -8,6 +8,8 @@ const schema = {
     body: Joi.object({
       createdAt: Joi.date().required(),
       activityId: activityId,
+      bringForwardDate: Joi.date().iso().allow(null),
+      bringForwardState: Joi.string().allow(null),
```

Comment:
Consider: If we are allowing null and expecting a value otherwise, this can be min(1) to disallow empty set.